README.md

```diff
@@ -503,6 +503,7 @@ You can:
 - Mahuton from Benin 🇧🇯 ([Mahuton](https://github.com/Mahuton))
 - Marco Aceti from Italy 🇮🇹 ([MarcoBuster](https://github.com/MarcoBuster))
 - Mariam Arab from Canada 🇨🇦 ([mariamarab](https://github.com/mariamarab))
+- Marek Fajkus from Czech Republic 🇨🇿 ([turboMack](https://github.com/turboMaCk))
```

Comment:
seems rebase added someone above me again? I hope it's not a big deal. Now I'm true off by one master....

Reply:
Don't worry!
We sort the list to avoid conflicts. 
The exact order doesn't matter